db/range_del_aggregator.cc

```diff
@@ -341,7 +426,12 @@ class CollapsedRangeDelMap : public RangeDelMap {
     }
   }
 
-  size_t Size() const { return rep_.size() - 1; }
+  size_t Size() const {
+    if (rep_.empty()) {
+      return 0;
```

Comment:
I should've reviewed this PR earlier. it has all the bug fixes we've been finding :(.